crates/holochain_integrity_types/CHANGELOG.md

```diff
@@ -7,8 +7,12 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## Unreleased
 
+- **BREAKING**: Renamed enum `ChainFilters` to `StopConditions` and renamed some of its variants.
```

Comment:
```suggestion
- **BREAKING**: Rename enum `ChainFilters` to `StopConditions` and rename some of its variants.
```

Reply:
The variant rename should be in the changelog.

Some people do update the individual changelogs but we're moving towards just maintaining all changes in the Holochain one. When we switch to the new release process, that's how it will work. So it's helpful to work that way now so we can better maintain our change history.

Reply:
Added variant changes in changelog. Still using this changelog for now and next time I'll use the main one,  if its okay?